socorro/processor/mozilla_transform_rules.py

```diff
@@ -1092,7 +1092,8 @@ def _predicate(self, raw_crash, raw_dumps, processed_crash, proc_meta):
         '''addons is a list of tuples containing (extension, version)'''
         addons = processed_crash.get('addons', [])
 
-        for extension, version in addons:
+        for addon in addons:
+            extension, version = addon.split(':', 1)
```

Comment:
If I'm "wrong" being nervous, then please add a comment here that explains why you can ALWAYS guarantee that each item in `addons` always contains a `:`.

Perhaps I just don't fully understand the context but the code doesn't look very defensive.